tests/conftest.py

```diff
@@ -171,31 +163,88 @@ def set_stream_messages(values):
     return async_mock
 
 
-@pytest.fixture(scope="function")
-def mock_setup_api_key(mocker):
-    mocker.patch("mentat.session.setup_api_key")
-    mocker.patch("mentat.conversation.is_model_available")
-    return
+class MockLlmApiHandler:
```

Comment:
```suggestion
class MockLLMApiHandler:
```

Reply:
I don't think you normally capitalize abbreviations in PascalCase; otherwise we'd be capitalizing both LLM and API and we'd have MockLLMAPIHandler. I think it's best this way because it's consistent

Reply:
Hmm yeah MockLLMAPIHandler is bad.

I looked at LangChain to see how they do it and they usually capitalize LLM and API in names: https://github.com/search?q=repo%3Alangchain-ai%2Flangchain+API+LLM&type=code

But yeah when they sequential `LLMAPI` is confusing so I guess leave it